docs/formats.md

```diff
@@ -167,6 +168,55 @@ BF               # map(*)
    FF            # primitive(*)
 ```
 
+### Ignoring unknown keys
```

Comment:
Should I move this section above the directly proceeding section (**Byte arrays and CBOR data types**) since it uses the same example CBOR data so would flow a bit nicer for the reader?